src/reports/package/combined-results-report.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import AccessibilityInsightsReport from './accessibilityInsightsReport';
+import { CombinedReportHtmlGenerator } from 'reports/combined-report-html-generator';
+import { ScanTimespan } from 'reports/components/report-sections/base-summary-report-section-props';
+import { ScanMetadata, ToolData } from 'common/types/store-data/unified-data-interface';
+
+export type CombinedResultsReportDeps = {
+    reportHtmlGenerator: CombinedReportHtmlGenerator;
+};
+
+export class CombinedResultsReport implements AccessibilityInsightsReport.Report {
+    constructor(
+        private readonly deps: CombinedResultsReportDeps,
+        private readonly parameters: AccessibilityInsightsReport.CombinedReportParameters,
+        private readonly toolInfo: ToolData,
+    ) {}
+
+    public asHTML(): string {
+        const { scanDetails } = this.parameters;
+
+        const targetAppInfo = {
+            name: scanDetails.basePageTitle,
+            url: scanDetails.baseUrl,
+        };
+
+        const scanMetadata: ScanMetadata = {
+            targetAppInfo: targetAppInfo,
+            toolData: this.toolInfo,
+            timestamp: null,
```

Comment:
It feels strange/redundant to set a `null` timestamp and then pass a second similar `timespan` parameter instead. Probably not a change for this PR, but I feel like it would be nice to combine those two (maybe replace `scanMetadata.timestamp` by putting the `timespan` in there instead and update the current `timestamp` usages to use `scanComplete` instead)